pkg/deploy/api/validation/validation.go

```diff
@@ -32,6 +35,11 @@ func ValidateDeploymentConfigUpdate(newConfig *deployapi.DeploymentConfig, oldCo
 	allErrs := fielderrors.ValidationErrorList{}
```

Comment:
I think there's a gap in my validation/persistence framework knowledge here... is `ValidateDeploymentConfigUpdate` called reflectively by the generic REST stuff?